src/common/BaseAnnotator.ts

```diff
@@ -151,4 +154,12 @@ export default class BaseAnnotator extends EventEmitter {
         this.store.dispatch<any>(store.fetchAnnotationsAction()); // eslint-disable-line @typescript-eslint/no-explicit-any
         this.store.dispatch<any>(store.fetchCollaboratorsAction()); // eslint-disable-line @typescript-eslint/no-explicit-any
     }
+
+    protected setLoaded(): void {
+        if (!this.loaded) {
+            this.emit(Event.ANNOTATIONS_READY, getAnnotations(this.store.getState()));
```

Comment:
Can we keep the terminology consistent throughout? How about `isInitialized` and `ANNOTATIONS_INIT`?

Reply:
Also, can we make the payload `{ annotations: getAnnotations(this.store.getState()) }` in case we need to add more to it later? It could help us limit breaking changes.

Reply:
Preview SDK already kind of has an event for when annotator is initialized -- this seems more like a `annotations_first_render` type of event?